concourse/pipelines/create-cloudfoundry.yml

```diff
@@ -1391,6 +1399,43 @@ jobs:
                   cf enable-feature-flag diego_docker
 
       - aggregate:
+        - task: deploy-compose-service-broker
+          config:
+            platform: linux
+            image_resource:
+              type: docker-image
+              source:
+                repository: governmentpaas/cf-cli
+            inputs:
+              - name: paas-cf
+              - name: paas-compose-broker
+              - name: config
+              - name: bosh-CA
+            run:
+              path: sh
+              args:
+                - -e
+                - -u
+                - -c
+                - |
+                  ./paas-cf/concourse/scripts/import_bosh_ca.sh
+
+                  . ./config/config.sh
+                  echo | cf login -a ${API_ENDPOINT} -u ${CF_ADMIN} -p ${CF_PASS}
+                  cf create-org service-brokers
+                  cf set-quota service-brokers small
+                  cf create-space compose -o service-brokers
+                  cf target -o service-brokers -s compose
+
+                  cp ./paas-cf/config/service-brokers/compose/catalog.json ./paas-compose-broker/
+                  cd paas-compose-broker/
+                  cf push --no-start
+                  cf set-env compose-broker BROKER_API_HOST 0.0.0.0
```

Comment:
As commented on the broker PR, it would be good for this to be the default behaviour making it unnecessary to specify this here.